src/coreclr/ilasm/asmparse.h

```diff
@@ -314,6 +314,7 @@ class AsmParse : public ErrorReporter
     friend char* nextBlank(_In_ __nullterminated char*);
     friend int ProcessEOF();
     friend unsigned __int8* skipType(unsigned __int8* ptr, BOOL fFixupType);
+    friend unsigned corCountArgs(BinStr* args);
```

Comment:
This is unnecessary. 
Instead, add a `static unsigned corCountArgs(BinStr* args);` declaration in grammar_before.cpp.